### PR TITLE
add CLF,CLZ as IMS motor types; add screen functions using motor-expe…

### PR DESCRIPTION
**Pull request by silkenelson**
…rt-screen

## Description
add CLF and CLZ to MMS as IMS motor types for factory faction
add screen() command (using motor-expert-screen command)

## Motivation and Context
start of working in issue #173 

## How Has This Been Tested?
opened motor screen for a few motors in an xcspython session

### CODE REVIEW

File: pcdsdevices/epics_motor.py [hunk 1]
```diff
@@ -479,6 +492,8 @@ def Motor(prefix, **kwargs):
     """
     # Available motor types
     motor_types = (('MMS', IMS),
+                   ('CLZ', IMS),
```
Comment (teddyrendahl) [inline pcdsdevices/epics_motor.py]: I believe that this is a duplicate commit?

https://github.com/pcdshub/pcdsdevices/blob/444f7a1a3a0ad978113ec0ed4c86a4bdec1f8ffd/pcdsdevices/epics_motor.py#L482

Think this was added in your last PR @silkenelson

File: pcdsdevices/epics_motor.py [hunk 2]
```diff
@@ -238,6 +240,17 @@ def _pos_changed(self, timestamp=None, old_value=None,
         super()._pos_changed(timestamp=timestamp, old_value=old_value,
                              value=value, **kwargs)
 
+    def screen(self):
+        """
+        Opens Epics motor expert screen for resetting motor after e.g. stalling
+        """
+        executable = 'motor-expert-screen'
+        if shutil.which(executable) is None:
+            print('%s is not on path, we cannot start the screen' % executable)
```
Comment (klauer) [inline pcdsdevices/epics_motor.py]:
```suggestion
            logger.error('%s is not on path, we cannot start the screen', executable)
```

**Comment by ZLLentz** [comment]
I love reading my own comments from 5 months ago that say "merging tomorrow" and then it isn't merged

**Comment by ZLLentz** [comment]
@klauer the most recent edit you did failed on flake8 line length violation

**Comment by klauer** [comment]
😬

File: pcdsdevices/epics_motor.py [hunk 3]
```diff
@@ -238,6 +240,17 @@ def _pos_changed(self, timestamp=None, old_value=None,
         super()._pos_changed(timestamp=timestamp, old_value=old_value,
                              value=value, **kwargs)
 
+    def screen(self):
+        """
+        Opens Epics motor expert screen for resetting motor after e.g. stalling
+        """
+        executable = 'motor-expert-screen'
+        if shutil.which(executable) is None:
+            logger.error('%s is not on path, we cannot start the screen', executable)
```
Comment (klauer) [inline pcdsdevices/epics_motor.py]:
```suggestion
            logger.error('%s is not on path, we cannot start the screen',
                         executable)
```

**Comment by ZLLentz** [comment]
For some reason I can't edit out the flake8 style error on this branch, so I'm just going to merge and fix the style error in the release notes PR

**Pull request merged**